src/routers/companyRouter.ts

```diff
@@ -8,6 +8,7 @@ import { logger } from "../lib/Logger";
 import FormValidator from "../utils/formValidator.util";
 import CompanyNumberSanitizer from "../utils/companyNumberSanitizer";
 import * as constants from "../constants/app.const";
```

Comment:
Should only be pulling in constants we only require - not everything ideally.

Reply:
done